src/renderer/components/modals/CreateMultisigAccount/components/SelectSignatories.tsx

```diff
@@ -109,6 +109,9 @@ export const SelectSignatories = ({ isActive, wallets, accounts, contacts, onSel
     });
   };
 
+  const isDisabled = (index: string, accountId: AccountId) =>
```

Comment:
A bit cleaner version with type and braces:
```suggestion
const isDisabled = (index: string, accountId: AccountId): boolean => {
    return (
      isSameItemSelected(index, selectedWallets[accountId]) || isSameItemSelected(index, selectedContacts[accountId])
    );
  };
```